backend/generated/prisma-client/index.js

```diff
@@ -1,54 +1,54 @@
-'use strict'
-Object.defineProperty(exports, '__esModule', { value: true })
-var prisma_lib_1 = require('prisma-client-lib')
-var typeDefs = require('./prisma-schema').typeDefs
+"use strict";
```

Comment:
These generated files should be outside eslint so this doesn't happen in the future